src/language-graphql/printer-graphql.js

```diff
@@ -633,12 +633,11 @@ function canAttachComment(node) {
 
 function printComment(commentPath) {
   const comment = commentPath.getValue();
-
-  switch (comment.kind) {
-    case "Comment":
-      return "#" + comment.value.trimRight();
-    default:
-      throw new Error("Not a comment: " + JSON.stringify(comment));
+  if(comment.kind === "Comment") {
+    return "#" + comment.value.trimRight();
+  }
+  else {
+    throw new Error("Not a comment: " + JSON.stringify(comment));
```

Comment:
IMO, the original `switch..case` is more readable.

Reply:
I think if there are only two conditions to be checked upon then "If" and "else" make more sense.

https://www.oreilly.com/library/view/high-performance-javascript/9781449382308/ch04s02.html